discoverer/src/main/java/eu/seaclouds/platform/discoverer/API/IAPI.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2014 SeaClouds
+ * Contact: SeaClouds
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package eu.seaclouds.platform.discoverer.API;
+
+
+public interface IAPI {
```

Comment:
It's just used inside the Discoverer to register api, it's not essential, but useful.